plugins/c9.ide.editors/editors.js

```diff
@@ -169,7 +175,27 @@ define(function(require, module, exports) {
                 return function cancel() { cancelled = true };
             }
         }
-        
+
+        function forbiddenFormat(ext, filename) {
+            // Obtain lst of excluded file formats
+            var lst = settings.get("user/tabs/@excludeFormats")
```

Comment:
A small note: it would be better to use unabbreviated name `list` instead of `lst` here, to match cloud9 coding style  https://github.com/c9/core/blob/master/docs/CODING_STANDARDS.md#variable-and-property-names